src/Akeneo/Pim/Enrichment/Component/Product/Validator/Constraints/AssociationTypeIsNotQuantified.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Akeneo\Pim\Enrichment\Component\Product\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+class AssociationTypeIsNotQuantified extends Constraint
+{
+    public const ASSOCIATION_TYPE_SHOULD_NOT_BE_QUANTIFIED_MESSAGE = 'pim_catalog.constraint.quantified_associations.association_type_should_not_be_quantified';
```

Comment:
Too be more consistent maybe it's more revelant
```suggestion
    public const ASSOCIATION_TYPE_SHOULD_NOT_BE_QUANTIFIED_MESSAGE = 'pim_catalog.constraint.associations.association_type_should_not_be_quantified';
```

Reply:
I hesitated about it but choose to keep it in `quantified_associations` because the error is only there because of QuantifiedAssociations. And there is no `pim_catalog.constraint.associations`